lamp-stack-ubuntu/README.md

```diff
@@ -0,0 +1,6 @@
+# Lamp stack on Ubuntu VM
```

Comment:
I will remove my lamp-stack-ubuntu template as there is no much difference with the similar one https://github.com/Azure/azure-quickstart-templates/tree/master/lamp-app